### ISSUE TITLE
Leveling tracker issues/suggestions

### ISSUE THREAD

**Issue by worldendDominator** [Lailloken/Lailloken-UI#288]
I've decided to try out the leveling tracker, and here's what I've found so far.

- Chamber of Sins L2 only has one layout provided, wile I think it can have several different ones in-game. I think it's not on you but on the layout creator though.
- Western Forest says to search the Weaver's Den "opposite of the waypoint". Is that actually correct? AFAIK, the den is opposite the side road leading to Alira (and the torch next to it near the main road), although there might indeed be waypoint correlation I never noticed.
- Mostly relevant to early acts: if the guide tells you to buy gems, but you have no currency or no spare sockets, you have to remember them or write them down. Is it feasible to add some input to quickly add the gem list to the notepad?

**Comment by worldendDominator** [comment]
Also in Western Forest, it guides you to get the Weaver first, leave, then return later for Alira and the blackguards. I think it's better to go for the blackguards first (straight way by the road), then the Weaver (easiest to find by following the wall, and after the blackguards you're at a wall).

**Comment by Lailloken** [comment]
> Chamber of Sins L2 only has one layout provided, wile I think it can have several different ones in-game. I think it's not on you but on the layout creator though.

Yeah, the guide the layouts are taken from is a bit outdated, but there aren't any newer ones (at least ones I could find). I just put them in because someone requested it.

> Western Forest says to search the Weaver's Den "opposite of the waypoint"

![road_opposite_the](https://github.com/Lailloken/Lailloken-UI/assets/61888437/82c98aca-0cc9-4851-a982-1ef2d79b4455)

I re-added these in v1.50.3 today. They used to be in v1.30, but I removed them temporarily because I knew some structural changes were coming to exile-leveling (the script checks the wording in the guide to know when to offer them). I think wording in the guide is confusing: "search the side of road opposite the waypoint".
Alira's path is on the same side as the waypoint, the Den is opposite of those.

> Mostly relevant to early acts: if the guide tells you to buy gems, but you have no currency or no spare sockets, you have to remember them or write them down. Is it feasible to add some input to quickly add the gem list to the notepad?

That's a good idea. I'll think about a way to implement this.

> Unless, of course, you're just copying the general campaign guide and have no input there. Disregard this then.

I used to do a little bit of tweaking to change the wording here and there (if my text-panel was getting too big at certain points, or to avoid confusion), but I had to rush implementation of the newer exile-leveling structure to get it ready for league-launch because I didn't receive a heads-up. But yeah, it's mostly just taking the steps as they are in exile-leveling.
